core/src/main/java/io/fabric8/maven/core/util/JibBuildServiceUtil.java

```diff
@@ -43,8 +43,8 @@
 import io.fabric8.maven.docker.access.AuthConfig;
 import io.fabric8.maven.docker.config.BuildImageConfiguration;
 import io.fabric8.maven.docker.service.RegistryService;
+import io.fabric8.maven.docker.util.EnvUtil;
```

Comment:
@rohanKanojia We should copy the prepareAbsoluteOutputDirPath method in fmp from dmp, while reflecting this PR in jshift.

Reply:
not sure what you mean, We already have this method in jkube/jshift https://github.com/jkubeio/jkube/blob/master/jkube-kit/common/src/main/java/io/jkube/kit/common/util/EnvUtil.java#L413